### PR TITLE
[OSDOCS-11001]: Moving HCP enable/disable docs from RHACM to OCP

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): 4.17

Issue: https://issues.redhat.com/browse/OSDOCS-11011

Link to docs preview: https://81100--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-prepare/hcp-enable-disable.html

QE review:
- [x] QE has approved this change.

NOTE FOR PEER REVIEWERS: This PR moves the enabling and disabling HCP content from the RHACM docs to the OCP docs. The content has not been changed since it was SME- and peer-reviewed by the RHACM team. This PR simply moves the content and applies OCP docs-specific formatting.

### CODE REVIEW

File: hosted_control_planes/hcp-prepare/hcp-enable-disable.adoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+:context: hcp-enable-disable
+
+toc::[]
+
+The {hcp} feature, as well as the `hypershift-addon` managed cluster add-on, are enabled by default. If you want to disable the feature, or if you disabled it and want to manually enable it, see the following procedures.
+
+include::modules/hcp-enable-manual.adoc[leveloffset=+1]
+include::modules/hcp-enable-manual-addon.adoc[leveloffset=+2]
+
+[id="hcp-disable"]
```
Comment (mburke5678) [inline hosted_control_planes/hcp-prepare/hcp-enable-disable.adoc:14]: Per the [doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-subsections-level-1-and-lower-headings):

```suggestion
[id="hcp-disable_{context}"]
```

**Comment by mburke5678** [comment]
@lahinson One comment. I did a quick scan of things and otherwise all looks fine.

**Comment by lahinson** [comment]
@mburke5678 Thank you, and good catch!

**Pull request merged**